doc/distrib/NodeHelpFiles/_review/Dynamo.Nodes.SelectFaces.md

```diff
@@ -0,0 +1,8 @@
+## In Depth
+`Select Faces` is a user interface node that provides selection of multiple faces in the current Revit document.
```

Comment:
Like the clarity available in the select edge node description regarding how user can interact with it. So suggest adding that for clarity here. "Select Faces is a user interface node that displays a cursor in Revit that allows you to select multiple faces in the current Revit document"